### ISSUE TITLE
Dropdown is not following high contrast color standards in Firefox browser

### ISSUE THREAD

**Issue by niadak** [OfficeDev/office-ui-fabric-react#3786]
### Bug Report

- __Package version(s)__: (fill this out)
- __Browser and OS versions__: (fill this out if relevant)

#### Priorities and help requested (not applicable if asking question):

Are you willing to submit a PR to fix? (Yes, No)

Requested priority: (Blocking, High, Normal, Low) 
Normal
Products/sites affected: (if applicable)
VSTS
#### Describe the issue:

Dropdown is not following high contrast color standards in Firefox browser

#### Actual behavior:

As per High Contrast Black Theme:
List items are White in color with blue background.
![image](https://user-images.githubusercontent.com/11937515/35326596-2f7faee0-011d-11e8-8b57-c75cf0b80919.png)

As per High Contrast White Theme:
List items are Black in color with violet background.
![image](https://user-images.githubusercontent.com/11937515/35326658-572c0e70-011d-11e8-9eca-1d403eeae6cd.png)

#### Expected behavior:

As per High Contrast Black Theme:
List items should be Black in color with blue background.
As per High Contrast White Theme:
List items should be White in color with violet background.

**Comment by niadak** [comment]
Focus is also not visible on the DropDown in Firefox High Contrast more.

**Comment by michaelangotti** [comment]
@niadak We only support IE and Edge for high contrast.
